exasol_sphinx_github_pages_generator/deployer.py

```diff
@@ -20,12 +35,21 @@ def __init__(self, source_dir, source_branch, current_commit_id, module_path,
         self.push_enabled = push_enabled
 
         self.worktree_paths = {"target_worktree": tempdir + "/worktrees/worktree_target",
-                              "source_worktree": tempdir + "/worktrees/worktree_source"}
+                               "source_worktree": tempdir + "/worktrees/worktree_source"}
         self.build_dir = tempdir + "/build"
         self.intermediate_dir = tempdir + "/intermediate"
 
     def check_out_source_branch_as_worktree(self,
                                             source_branch_exists_locally):
```

Comment:
Add type hint

Reply:
I don't know where exactly you are missing a type hint here?

Reply:
def check_out_source_branch_as_worktree(self,
                                            source_branch_exists_locally: int) -> None:

Reply:
Ah, yes.